index.bs

```diff
@@ -1298,6 +1321,14 @@ To <dfn>match an attribution source's filter data against filters</dfn> given an
         </dl>
 1. Return true.
 
+To <dfn>match an attribution source's filter data against filters</dfn> given an
```

Comment:
CC @johnivdel to verify that the `OR` logic matches what is expected/intended for negated filters.

Reply:
Ping @johnivdel